docs/source/releasenotes.rst

```diff
@@ -2,41 +2,32 @@
 Release notes
 =============
 
+The `pavics-sdi` repository only holds the documentation. Although all the individual components of the PAVICS platform are open-source, the code configuring, deploying and linking these components together is in a private repository.  We are planning to remove all sensitive material from this private repository so it can be made public. In the meantime, these release notes are the public facing information tracking the development of the platform.
+
 
 PAVICS-SDI
 ==========
 
+1.2.x (2019-06-17)
```

Comment:
@huard 

I haven't been tagging since we moved to a continuous deployment style.

But if it helps to have a familiar vocabulary to talk to other users/stakeholders, I can keep tagging.  Let me know.

Reply:
Yes, I think it's useful to use semantic versioning for the platform to keep track of added functionality, bug fixes and so on. 

Also, ideally, the version for the docs would be the same as the version for PAVICS. Since they are in two different repos, this is not very convenient to do. Hopefully we can find a way to have the docker-compose and the docs at the same place so they stay in sync.

Reply:
> Yes, I think it's useful to use semantic versioning for the platform to keep track of added functionality, bug fixes and so on.
> 

OK will tag from now on.

> Also, ideally, the version for the docs would be the same as the version for PAVICS. Since they are in two different repos, this is not very convenient to do. Hopefully we can find a way to have the docker-compose and the docs at the same place so they stay in sync.

docker-compose and the docs at the same place --> merge PAVICS and pavics-sdi repo? I'd say rather just host the release info in the PAVICS repo and use interphinx to link it to pavics-sdi ... this will require a public PAVICS repo ...